src/service/team.js

```diff
@@ -0,0 +1,23 @@
+import { addItem } from './firebase'
+import { isEmpty, isNil, where } from 'ramda'
+
+const INVALID_TEAM_ERROR = 'you are trying to save an invalid team'
+const TEAM_COLLECTION = 'teams'
```

Comment:
the strings for errors are some in english and some in spanish

Reply:
hahahaha true, not sure how this should be now... not all of those strings are going to be in the screen...